src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -120,6 +122,8 @@ class LocalBackend extends Backend with LazyLogging {
     } else {
       if (rc == 0) {
         evaluateCallOutputs(workflowDescriptor, call, hostAbsoluteFilePath, localEngineFunctions, scopedLookupFunction, interpolateStrings=true)
+      } else if (rc == 143) {
```

Comment:
I agree with that, I was assuming such a constant already existed but if not then just a comment would be preferable.